Source/Core/BoxGeometry.js

```diff
@@ -124,7 +125,7 @@ define([
      *
      * @exception {DeveloperError} AxisAlignedBoundingBox must be defined.
      *
-     * @see BoxGeometry.createGeometry
+     * 
```

Comment:
Do not include this extra line of whitespace, there should just be one line of whitespace between `@exception` and `@example`.